projects/packages/forms/src/contact-form/css/grunion.scss

```diff
@@ -1307,3 +1310,22 @@ on production builds, the attributes are being reordered, causing side-effects
 .contact-form-ajax-submission:not(.submission-success) {
 	display: none;
 }
+
+.jetpack-form-submit-button button.is-submitting::after {
+	content: "";
+	display: inline-block;
+	width: 1em;
+	height: 1em;
+	margin-left: 0.5em;
+	border: 2px solid currentColor;
+	border-right-color: transparent;
+	border-radius: 50%;
+	animation: jp-spinner 0.75s linear infinite;
+}
+
+@keyframes jp-spinner {
```

Comment:
Not a biggie, but `jp-forms-spinner` would be more specific to avoid colliding with any other Jetpack feature.